tasks/pdagent.yml

```diff
@@ -0,0 +1,19 @@
+# Install pdagent on a master
```

Comment:
I'm a bit hesitant about increasing the scope/complexity of this role to also manage the `pdagent`. For some context, I'm already uneasy that we also install RabbitMQ and Redis instead of having folks use a solid role dedicated to solely managing those services. 

Personally, I'd keep this as a dedicated role that would be applied to the server(s) that need `pdagent`. That said, if the community thinks this is a sane and valuable addition to this role, I'm open to keeping it.

Reply:
I am in agreement I see no reason we should install the `pd_agent` (there is already a pagerduty plugin which has much better integration with sensu) at all let alone in a role. I am ok with it being an optional thing with a default of false. I really have no clue what i am talking about from a ansible perspective but I would say the same thing in chef.